protos/protos.go

```diff
@@ -126,7 +126,7 @@ func (p Protocol) String() string {
 }
 
 type Protocols interface {
-	BpfFilter(with_vlans bool) string
+	BpfFilter(with_vlans bool, with_icmp bool) string
```

Comment:
Ok. I was already wondering what the naming convention is in the project but didn't find a consistent one.

But inside the `Config` structs you still go with Camel_case so that it's underscores in the .yml config file?